clients/lsp-markdown.el

```diff
@@ -40,23 +40,62 @@
   :package-version '(lsp-mode . "7.1"))
 
 (defcustom lsp-markdown-server-command-args '("--parser=remark-parse" "--stdio")
-  "commandline-arguments for the markdown lsp server."
+  "Command-line arguments for the markdown lsp server."
   :type '(repeat 'string)
   :group 'lsp-markdown
   :package-version '(lsp-mode . "7.1"))
 
+(defcustom lsp-markdown-remark-plugins [["#remark-preset-lint-markdown-style-guide"]]
+  "The JSON configuration object for plugins.
+
+For a complete list of plugins, check:
+ https://github.com/unifiedjs/unified-language-server/blob/main/CONFIGURATION.md#re-using-settings"
+  :type 'lsp-string-vector
+  :group 'lsp-markdown
+  :package-version '(lsp-mode . "7.1"))
+
+(defcustom lsp-markdown-remark-check-text-with-setting "retext-english"
+  "Configure `checkTextWith' subproperty.
+
+For a complete list of plugins, check:
+ https://github.com/unifiedjs/unified-language-server/blob/main/CONFIGURATION.md#re-using-settings"
+  :type '(choice (
```

Comment:
Please not that the `:type` here was a rough copy from some online discussion. It might not be correct. Doing `M-x customize-group RET lsp-markdown` did not show this property.